<launch>
  <arg name="hand_serial" default="1082"/>
  <include file="$(find sr_robot_launch)/launch/sr_right_ur10arm_hand.launch" >
    <arg name="hand_serial" value="$(arg hand_serial)"/>
  </include>
  <node name="graspatron_demo1" pkg="sr_graspatron" type="demo1.py" output="screen">
      <param name="initial_state_name" type="string" value="initial_state" />
  </node>
</launch>
